cob_moveit_bringup/launch/move_group.launch

```diff
@@ -36,8 +37,9 @@
 
   <!-- Trajectory Execution Functionality -->
   <include ns="move_group" file="$(find cob_moveit_bringup)/launch/controller_manager.xml" if="$(arg allow_trajectory_execution)">
-    <arg name="config_path" value="$(arg config_path)"/>
+    <arg name="moveit_controller_manager" value="fake" if="$(arg fake_execution)"/>
```

Comment:
why do you need this?
isn't the line `<arg name="fake_execution" value="$(arg fake_execution)"/>` sufficient for fake_execution?